src/unitxt/catalog/cards/piqa.json

```diff
@@ -98,9 +98,17 @@
             "to_field": "numbers",
             "use_query": true
         },
+        {
+            "type": "add_constant",
+            "field": "choices_list",
+            "to_field": "choices_list",
+            "add": [
+                ""
+            ]
+        },
         {
             "type": "join_str",
-            "separator": " ",
+            "separator": "\n",
```

Comment:
Why this change?  Does anyone use this dataset?

Reply:
The change was deleted and will be resolved in a separate PR